Automattic/wp-calypso
# Domain management revamp: Update single domain page

Related to #96812

## Proposed Changes

This PR updates the looks of the single domain overview section following the design - ePfz53j71KKDLWtuSm567A-fi-4948_7391.

There are slight deviations from the design to maintain consistency with the wordpress.com/sites screen. Please keep that in mind when comparing.

## Why are these changes being made?


* This is part of the domain management revamp project - pfuQfP-13x-p2.

## Testing Instructions



* Go to http://calypso.localhost:3000/domains/manage
* Enable the feature flag by entering window.sessionStorage.setItem('flags', 'calypso/all-domain-management') in the - browser console. Reload the page.
* Click on a few domains and make sure the overview is working.
* Make sure the overview section matches the design and it looks good on mobile.

## Screenshots

![image](https://github.com/user-attachments/assets/5c86cc26-caf7-4d63-8bd6-030667f105f8)

## Pre-merge Checklist
